core/server/master/src/main/java/alluxio/master/metrics/DefaultMetricsMaster.java

```diff
@@ -53,8 +51,10 @@
  */
 public class DefaultMetricsMaster extends CoreMaster implements MetricsMaster, NoopJournaled {
   private static final Logger LOG = LoggerFactory.getLogger(DefaultMetricsMaster.class);
-  private final Set<MultiValueMetricsAggregator> mMultiValueMetricsAggregatorRegistry =
-      new HashSet<>();
+  // A map from the to be aggregated metric name to aggregator itself
+  // This registry only holds aggregator for master metrics
+  private final Map<String, MultiValueMetricsAggregator>
+      mAggregatorRegistry = new HashMap<>();
```

Comment:
nit: This seems like an odd line break. Does it not fit on one line?

Reply:
Fixed, thanks